### DISCUSSION TITLE
Original post in a discussion should have a "reply" button at the top

### DISCUSSION THREAD

**Discussion by timothyis** [community/community#38072] [Discussions]
When you visit a discussion that has hundreds of replies, it's quite irritating to have to scroll for ages to reach the bottom where the text field exists to post a reply. Please see https://github.com/vercel/next.js/discussions/41745 as an example.

In my opinion, the bottom right of the discussion original post box should have a "reply" button to the right of the reactions: 
<img width="931" alt="image" src="https://user-images.githubusercontent.com/1695613/199523831-3cb13552-e015-40be-8f6d-68bcb361b0a0.png">

**Comment by ossanna16** [comment]
Hi there @timothyis and welcome to our community! Thank you for this feature request!
